client/src/api/block_builder/mod.rs

```diff
@@ -81,8 +84,8 @@ pub struct ClientBlockBuilderOptions {
     pub data: Option<String>,
     /// Parents
     pub parents: Option<Vec<BlockId>>,
-    /// Allow burning of native tokens
-    pub allow_burning: Option<bool>,
+    /// Explicit burning of aliases, nfts, foundries and native tokens
+    pub burn: Option<Burn>,
```

Comment:
This also needs to be updated in the bindings

Reply:
Done